lib/connect.c

```diff
@@ -1550,6 +1550,24 @@ bool Curl_connalive(struct connectdata *conn)
 int Curl_closesocket(struct Curl_easy *data, struct connectdata *conn,
                      curl_socket_t sock)
 {
+#ifdef MPTCP
+  struct mptcp_info inf;
+  socklen_t optlen;
+  int ret;
+  if(data->set.verbose &&
+     conn->transport == TRNSPRT_TCP && conn->bits.tcp_multipath) {
+    optlen = sizeof(inf);
+    if(!getsockopt(sock, SOL_MPTCP, MPTCP_INFO, &inf, &optlen)) {
+      infof(data,"Multipath TCP used %d subflows on this connection",inf.mptcpi_subflows);
+    }
+    else {
+      infof(data,"Multipath TCP was DISABLED on this connection");
+    }
+  }
+#endif /* MPTCP */
+
+
+
```

Comment:
This block looks like it's there for debugging? Maybe use `#ifdef DEBUGBUILD` as well?

Reply:
Yes, good suggestion